### PR TITLE
bluetoothconnector 2.0.0

### PR DESCRIPTION
**Pull request by lapfelix**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bluetoothconnector.rb [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
+  sha256 "41474f185fd40602fb197e79df5cd4783ff57b92c1dfe2b8e2c4661af038ed9b"
+  head "https://github.com/lapfelix/BluetoothConnector.git"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "b49cd1726285c91d42935e0052814940a9c33a99df54ea1da2eab4c5b7411c71" => :mojave
-    sha256 "b49cd1726285c91d42935e0052814940a9c33a99df54ea1da2eab4c5b7411c71" => :high_sierra
+    sha256 "1c3b876a95fe14f0716239d9eadf882b40ce9d0df71080502581800231904340" => :mojave
+    sha256 "282773ff899326dc1c0dfc1fa7941da80c46d314ae1b0930a68d7e3e5758a73b" => :high_sierra
   end
```
Comment (chenrui333) [inline Formula/bluetoothconnector.rb:12]: Why did mojave and high_sierra was pointing to the same hash before?

Ideally, you dont have to touch the bottle block. I am curious why you need to do this.

Comment (lapfelix) [reply Formula/bluetoothconnector.rb:12]: Hmmmm I did not manually edit this. I'll investigate to see what's going on.

Comment (lapfelix) [reply Formula/bluetoothconnector.rb:12]: Okay so I first started this PR with `brew bump-formula-pr` which made this edit: https://github.com/Homebrew/homebrew-core/pull/54178/files

So, after reading up on these hashes, it seems like they'll be edited by the CI once it gets merged anyway? So I assume it's okay to keep them as is? I can revert them back to the old hash if it makes things simpler.

Comment (SMillerDev) [reply Formula/bluetoothconnector.rb:12]: I think the new values will conflict, please change them back.

Comment (chenrui333) [reply Formula/bluetoothconnector.rb:12]: @lapfelix the CI would take care of the bottle block, no need to update it manually.

Comment (lapfelix) [reply Formula/bluetoothconnector.rb:12]: @SMillerDev I changed them back

**Comment by SMillerDev** [comment]
Thanks @lapfelix  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**